sonaranalyzer-dotnet/src/SonarAnalyzer.VisualBasic/Rules/MethodOverloadsShouldBeGrouped.cs

```diff
@@ -82,6 +82,25 @@ private bool IsStaticStatement(MethodBaseSyntax statement)
         {
             return statement.DescendantTokens().Any(x => x.Kind() == SyntaxKind.SharedKeyword);
         }
+        
+        protected override bool IsAbstract(StatementSyntax member)
```

Comment:
According to what we decided as coding style, these 2 methods are not in the apporpriate place due to their visibility accessors